TagsterWebAPI/src/Tagster.Application/Commands/ChangeUserRole/ChangeUserRole.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Tagster.Application.Commands.ChangeUserRole;
+
+public class ChangeUserRole
```

Comment:
where I can find handler for this command?

Reply:
I forgot add it. But now it is in repository